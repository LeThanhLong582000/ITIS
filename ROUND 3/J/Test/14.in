84
367592253
560531229
131226304
662461833
824317407
870976226
683679309
144423756
438913621
567360533
262367910
896825254
726757461
384137403
996758739
907854912
118065914
941363228
774956423
981270624
115383218
472199040
412362062
114789120
804402987
407004688
468814539
226185538
747665713
466677167
185035327
979861851
490085432
929600450
288645147
136426858
540443611
921703345
944549991
509483447
880744435
494469018
496816323
528803682
974338980
168470701
54232610
973885443
959517294
472041099
138538132
840569945
393969519
602062999
920448732
922602809
496385765
18748338
410337373
388150066
924689774
248529868
271795028
895582728
511526482
420943282
877223457
381663800
214771036
959746785
506321401
672026724
304883083
133402233
232375376
522723692
503793492
130710777
897178506
185599991
237214366
742977283
305926493
473257674
