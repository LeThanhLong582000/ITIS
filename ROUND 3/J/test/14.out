Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Shin
Lin
Lin
Lin
Shin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Shin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Lin
Shin
Lin
Lin
Lin
